checker/jdk/index/src/java/io/File.java

```diff
@@ -195,7 +195,7 @@ int getPrefixLength() {
      * string for convenience.  This string contains a single character, namely
      * <code>{@link #separatorChar}</code>.
      */
-    public static final @MinLen(1) String separator = "" + separatorChar;
+    public static final /*!MinLen(1)*/ String separator = "" + separatorChar;
```

Comment:
That's going to be a problem when we want to add String support. Since the annotation is commented out, it's not a problem now - but if you'd prefer, I can remove it entirely.